util/failureExpanded.js

```diff
@@ -8,4 +8,4 @@ const failureExpanded = (errors) => {
   })
 }
 
-module.exports = failureExpanded
\ No newline at end of file
+module.exports = failureExpanded
```

Comment:
Noticed a few of these files have the newline at end of file removed. I think this is a pattern to have them in BK, so maybe we should keep them there?

Reply:
This PR actually adds new line, that wasn't there originally. But I'll reset the changes, as it's not related. We need to setup linter for test-collector-javascript